copis/core.py

```diff
@@ -331,7 +345,7 @@ def send_now(self, command):
         else:
             logging.error("Not connected to device.")
 
-    def _image(self, resuming=False) -> None:
+    def _image(self) -> None:
```

Comment:
Keep `resuming` (it should do something that will be added in the future)

Reply:
Then let's add it back in the future.